manage.py

```diff
@@ -77,8 +78,23 @@ def setup_prod():
 
 
 def setup_general():
-    """Runs the set-up needed for both local development and production."""
+    """Runs the set-up needed for both local development and production.
+       Also sets up first admin user."""
     Role.insert_roles()
+    admin_query = Role.query.filter_by(name='Administrator')
+    if admin_query.first() is not None:
+        admin_role_id = admin_query.first().id
+        if User.query.filter_by(email=Config.ADMIN_EMAIL).first() is None:
+            user = User(
+                first_name="Admin",
+                last_name="Account",
+                password=Config.ADMIN_PASSWORD,
+                confirmed=True,
+                email=Config.ADMIN_EMAIL,
+                role_id=admin_role_id)
```

Comment:
Removed the role_id setting since User init will automatically assign the role of an account to administrator if the account email matches that of the config's admin email.